tests/lib/prepare-utils.sh

```diff
@@ -137,15 +137,21 @@ install_core22_deps() {
     sudo snap install ubuntu-image --classic
 }
 
-download_core22_snaps() {
+download_core24_snaps() {
     local snap_branch="$1"
 
     # get the model
-    curl -o ubuntu-core-amd64-dangerous.model https://raw.githubusercontent.com/snapcore/models/master/ubuntu-core-22-amd64-dangerous.model
+    #curl -o ubuntu-core-amd64-dangerous.model https://raw.githubusercontent.com/snapcore/models/master/ubuntu-core-24-amd64-dangerous.model
+    cp "${PROJECT_PATH}/ubuntu-core-24-amd64-dangerous.model" ubuntu-core-amd64-dangerous.model
 
     # download neccessary images
     snap download pc-kernel --channel=22/${snap_branch} --basename=upstream-pc-kernel
     snap download pc --channel=22/${snap_branch} --basename=upstream-pc-gadget
+    unsquashfs -d upstream-pc-gadget upstream-pc-gadget.snap
```

Comment:
Please add TODO for when the 24 channel is available

Reply:
Done.